metric/prometheus.go

```diff
@@ -90,12 +109,29 @@ func NewGauge(namespace, subsystem, name, metric string,
 	return gaugeMap[metricName]
 }
 
+func getAndValid(namespace, subsystem, name, metric string, labels []string) (string, error) {
+	metricName := GetName(namespace, subsystem, name, metric)
+	if ValidMetricName(metricName) == false {
```

Comment:
`ValidMetricName` name means return true if it is a valid name; prefer:
```
if !ValidMetricName(metricName) {
   
}
```

Reply:
Sorry, cannot accept this comment, because this is not easy for reading.